Sandbox/src/Game/Commands/BuryCommand.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include "Game/Commands/ICommand.h"
+#include "Game/Player/Player.h"
+
+class BuryCommand : public ICommand
+{
+private:
+	std::shared_ptr<Player> mPlayer;
+
+	bool CanBury(int itemIndex)
+	{
+		switch (itemIndex)
+		{
+		case 10: //bones
```

Comment:
would also like this in a constant variable somewhere

Reply:
This code is temporary, this will definitely be read from a file or stored as constants in a concrete  class implementation somewhere.